examples/va/nvidia-vfio-passthrough/edpm/nodeset/values.yaml

```diff
@@ -0,0 +1,217 @@
+# yamllint disable rule:line-length
+# local-config: referenced, but not emitted by kustomize
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: edpm-nodeset-values
+  annotations:
+    config.kubernetes.io/local-config: "true"
+data:
+  # Mapped in va/nvidia-vfio-passthrough/edpm/nodeset/kustomization.yaml to baremetalset-password-secret.data.NodeRootPassword
+  root_password: cmVkaGF0Cg==
+  # Mapped in va/nvidia-vfio-passthrough/edpm/nodeset/kustomization.yaml to OpenStackDataPlaneNodeSet.spec.preProvisioned
+  preProvisioned: false
+  # Mapped in va/nvidia-vfio-passthrough/edpm/nodeset/kustomization.yaml to BareMetalHost.metadata.annotations.inspect.metal3.io
+  metal3_inspection: ""
```

Comment:
Recommend `disabled` here as well unless you have a specific need for it

Reply:
What is BMO recommendation for doing ispection or not, by default? @hjensas  WDYT?